src/scripts/main.js

```diff
@@ -1,3 +1,30 @@
 'use strict';
 
-// write code here
+function sortList(employees) {
+  const sorted = employees.sort((firstElement, secondElement) => {
+    return (converter(secondElement.getAttribute('data-salary'))
+      - converter(firstElement.getAttribute('data-salary')));
+  });
+
+  document.body.children[1].append(...sorted);
+}
+
+function getEmployees(arrayEmployees) {
+  return arrayEmployees.map(employee => {
+    return {
+      name: employee.innerHTML.trim(),
+      position: employee.getAttribute('data-position'),
+      salary: employee.getAttribute('data-salary'),
+      age: employee.getAttribute('data-age'),
+    };
+  });
+}
+
+function converter(salary) {
+  return salary.replaceAll(',', '').replace('$', '');
+}
+
+const worker = [...document.body.children[1].children];
```

Comment:
Would be better to access workers via getting ul element and its children, rather then just children chain. This way it's more clear what you are looking for
However it's not crucial

Reply:
@ViktorHavryliuk Done.